group_vars/researchdata/researchdata_staging.yml

```diff
@@ -0,0 +1,6 @@
+postgres_host: "lib-postgres3.princeton.edu"
```

Comment:
no interest in lib-postgres-staging1.princeton.edu?

Reply:
@kayiwa I copied from another role and did not try that.  Let me see if it will work...